### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by stolarczyk**
**to do:**
- [x] release date
- [x] version

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
 # Changelog
 
+## [0.4.9] - unreleased
+### Added
+- add `--version` argument in `VersionInHelpParser`
+
```
Comment (nsheff) [inline docs/changelog.md:8]: isn't this in there automatically?

File: ubiquerg/cli_tools.py [hunk 2]
```diff
@@ -16,6 +16,8 @@ def __init__(self, version=None, **kwargs):
         """ Overwrites the inherited init. Saves the version as an object attribute for further use. """
         super(VersionInHelpParser, self).__init__(**kwargs)
         self.version = version
+        if self.version is not None:
+            self.add_argument('--version', action='version', version='%(prog)s {}'.format(self.version))
```
Comment (nsheff) [inline ubiquerg/cli_tools.py:20]: I guess this is just for python 2 ? I think ti's built-in in python3

**Review by nsheff — APPROVED** [review]
This is fine with me, let's release it.

File: ubiquerg/cli_tools.py (hunk 2 above)
Comment (stolarczyk) [reply ubiquerg/cli_tools.py:20]: I think it did not work for either. They advertise the `version` action for Python3: https://docs.python.org/3.7/library/argparse.html#action

**Pull request merged**